OpenRA.Mods.RA/Effects/GpsDot.cs

```diff
@@ -111,7 +111,7 @@ bool IsTargetableBy(Player toPlayer, out bool shouldRenderIndicator)
 
 			shouldRenderIndicator = !f2.HasRenderables;
 
-			return f2.Visible && !f2.Shrouded;
+			return f2.Visible && !f2.Shrouded && toPlayer.Shroud.IsExplored(self.CenterPosition);
```

Comment:
It would be better to use `!world.ShroudObscures()` to make sure that you are synced with the player's render state.